### ISSUE TITLE
Bluetooth low energy

### ISSUE THREAD

**Issue by rdaguerre** [juamiso/ANT_BMS#6]
Hi, excuse me for muy english.
Can you use ble antbms communication? 
The problem with serial port style is that only connect with bms from python and can not work with the aplication both together. 
Thanks very much.

**Comment by juamiso** [comment]
Yes I know, you either have BMS connected to raspy via Python or to the android/iPhone app. I can not find a workaround for this. If you can, please let me know.

**Comment by Carlosfandango22** [comment]
Hello, 
advanced apologies for my complete ignorance. 

What do I need to do to run this? I have a raspberry pi 3, but I literally have no idea how to use this. I come from C++ background so I am completely new. 

Thanks.
